pulpcore/cli/file/content.py

```diff
@@ -92,10 +93,14 @@ def content(ctx: click.Context, pulp_ctx: PulpCLIContext, content_type: str) ->
     click.option(
         "--sha256",
         "artifact",
-        required=True,
-        help=_("Digest of the artifact to use"),
+        help=_("Digest of the artifact to use [deprecated]"),
```

Comment:
Is it time for another round of brainstorming over the "manage the content in a repository" workflows?